### PR TITLE
fix copy button issue & others minor fixes

### PR DESCRIPTION
**Pull request by masskoder**
This PR addresses the problem with the copy button. Therefore the copy button will be visible on the initial tab. Additionally, an initial solution has been implemented to hide the default scrollbar of Hugo highlight.

https://www.awesomescreenshot.com/video/18764983?key=80e643d0bffed2940f245cbbaf210426

### CODE REVIEW

File: layouts/shortcodes/data-playground-code-tabs.html
Comment (daniellemaxwell) [inline layouts/shortcodes/data-playground-code-tabs.html:19]: Should the `id` be removed if it's no longer used in the `addCopyButtons` function?

File: static/js/copy-code-button.js
Comment (daniellemaxwell) [inline static/js/copy-code-button.js:10]: Could you explain the decision for replacing `textContent` with `innerText`? I'm just curious.

File: content/en/data-playground/dc-metro.md
Comment (daniellemaxwell) [inline content/en/data-playground/dc-metro.md:36]: This was removed in a PR last week. Could we agree to make sure a branch is updated with changes from main when creating PRs? I understand that changes can be merged while a PR is waiting for review and that we can create PRs around the same time. However, I'm thinking more about changes that occur a day or more before a PR is created.

**Review by daniellemaxwell — COMMENTED** [review]
The code looks good and the copy button looks great! Once the merge conflicts are resolved, I think this will be good to go.

**Review by daniellemaxwell — APPROVED** [review]
Thanks resolving the conflicts! Before merging would you mind double checking the position of the copy buttons? In the video included for the acceptance criteria, the positioning looks fine. However, on my screen it appears to be on the edge.

Screenshot 2023-06-30 at 10 36 39 AM

Screenshot 2023-06-30 at 10 36 47 AM

File: static/js/copy-code-button.js
Comment (masskoder) [reply static/js/copy-code-button.js:10]:
```
- textContent is not aware of styles. So it returns the entire content
- innerText is aware of CSS.
```

**Comment by masskoder** [comment]
> Thanks resolving the conflicts! Before merging would you mind double checking the position of the copy buttons? In the video included for the acceptance criteria, the positioning looks fine. However, on my screen it appears to be on the edge.
> 
> Screenshot 2023-06-30 at 10 36 39 AM Screenshot 2023-06-30 at 10 36 47 AM

Maybe it's CSS cache issue , in my side, it's working properly

**Pull request merged**
